.editorconfig

```diff
@@ -0,0 +1,12 @@
+# https://editorconfig.org
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+insert_final_newline = true
+
+[*.{js,css,htm,html}]
+charset = utf-8
+trim_trailing_whitespace = true
```

Comment:
What about adding a trim for leading spaces? Leading spaces can prevent emails being sent to users if captured in an email form field.

Reply:
This is just for the editor, whenever you save a file with vscode, sublime, etc it will auto-format the files based on these settings. This would just affect source code and not any form input.

Reply:
Okay, cool. Thank you!